desktop_version/src/editor.cpp

```diff
@@ -1959,10 +1959,21 @@ bool editorclass::load(std::string& _path)
 
         if (pKey == "script")
         {
+            // load in the linebreak attribute
+            linebreak = "";
+            pElem->QueryStringAttribute("linebreak", &linebreak);
+            char splitvalue = linebreak[0];
+            // "" is the only value where splitvalue and linebreak are different
+            if (std::strcmp(linebreak, "") == 0)
+            {
+                splitvalue = '|'; // To keep compatibility with older levels
+                linebreak = "\n"; // So this conversion only happens once
```

Comment:
I'm not sure if existing levels should be automatically converted to using newlines. Certainly newly-created levels should be using newlines, but there could be a level out there retexturing newline characters in `font.png` and using them in dialogue. I know for certain I've done it in the title of one of my levels, Alias Space: Checkpoint Abuse.